site/docs/.vuepress/styles/_select.scss

```diff
@@ -0,0 +1,19 @@
+.action-button,
+.navbar-item,
+.site-name,
+.logo,
+img[alt="grammY"],
+.next,
+.prev {
+  user-select: none;
+}
```

Comment:
```suggestion
img[alt="grammY"],
.badge,
.actions,
.navbar,
.sidebar,
.footer, // footer on homescreen
footer, // footer on page
nav {
  user-select: none;
}
```

Reply:
Btw, why is only the `img[alt="grammY"]` excluded, instead of all images?

Reply:
Good point. I was just doubtful.